multi-model-endpoint/deploy-endpoint.yml

```diff
@@ -39,6 +39,11 @@ Resources:
       StageName:
         Ref: Stage
       Name: !Sub ${ModelName}-api-${Stage}
+      Domain: 
+        DomainName: api.dev.detektivkollektiv.org
```

Comment:
You would have to make the Stage dynamic in order for the deployment to work on qa/prod as well